### ISSUE TITLE
Inference

### ISSUE THREAD

**Issue by AntonioArtur** [kkoutini/passt_hear21#1]
How do i get class predictions instead of embeddings?

**Comment by kkoutini** [comment]
Hi, 
I've added this feature in a new release 0.0.9, please update using:
```
pip install -e 'git+https://github.com/kkoutini/passt_hear21@0.0.9#egg=hear21passt' 
```
After that you can get the logits  (before the sigmoid activation) for the 527 classes, if you load the model like this:
```
model = load_model(mode="logits").cuda()
```

**Comment by faroit** [comment]
@kkoutini this is super useful. Maybe add this to the readme

**Comment by faroit** [comment]
@kkoutini is there a way to also load the ESC-50 pretrained models with `passt_hear21`?

**Comment by kkoutini** [comment]
> @kkoutini is there a way to also load the ESC-50 pretrained models with `passt_hear21`?

@faroit I added more details for fine-tuning and loading the pre-trained models on ESC-50 [here](https://github.com/kkoutini/PaSST/tree/main/esc50) and the pre-trained models can be downloaded [here](https://github.com/kkoutini/PaSST/releases/tag/v.0.0.6).

**Comment by faroit** [comment]
@kkoutini Thanks!
